spec/requests/v2/assessments_controller_spec.rb

```diff
@@ -832,8 +832,8 @@ def cash_transactions(amount)
 
           it "has disposable income" do
             expect(assessment.fetch(:disposable_income)).to eq(
-              { monthly_equivalents: { all_sources: { child_care: 0.0, rent_or_mortgage: 234.33, maintenance_out: 333.07, legal_aid: 0.0 },
-                                       bank_transactions: { child_care: 0.0, rent_or_mortgage: 234.33, maintenance_out: 333.07, legal_aid: 0.0 },
```

Comment:
This test change indicates an introduced defect around the complex 'housing costs' code